index.js

```diff
@@ -9,4 +9,9 @@ const server = http.createServer((req, res) => {
 
 server.listen(port, () => {
   console.log(`Server running on http://localhost:${port}/`);
-});
\ No newline at end of file
+});
+
+// Error handling
+server.on('error', (err) => {
+  console.error('Server error:', err);
+});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [16-18]

LGTM!


<!-- This is an auto-generated comment by OSS CodeRabbit -->